eng/testing/tests.mobile.targets

```diff
@@ -146,6 +158,28 @@
 
   </Target>
 
+  <Target Condition="'$(EnableAggressiveTrimming)' == 'true'"
+          Name="ConfigureTrimming"
```

Comment:
Can we move `Name` to be the first attribute? it will make it easier to read.

Reply:
Yes, also condensing it to be all in one line.